api/pkg/service/auth/auth.go

```diff
@@ -123,7 +123,7 @@ func (r *request) addUser(ghUser *github.User) (*model.User, error) {
 
 	// Check if user exist
 	q := r.db.Model(&model.User{}).
-		Where("LOWER(github_login) = ?", strings.ToLower(ghUser.GetLogin()))
+		Where("LOWER(github_login) = ? or github_login = ?", strings.ToLower(ghUser.GetLogin()), ghUser.GetLogin())
```

Comment:
here in the first condition we are converting to lowercase and then checking
`LOWER(github_login) = ?", strings.ToLower(ghUser.GetLogin())`
so would we require the 2nd check after `or` ?
first one will cover all cases I think as it is converting to lower case both the words
wdyt?

Reply:
@sm43 Thanks. Yeah. That's make sense. Sorry, it was added as part of initial plan. It's not required anymore. TBC, this logic `LOWER(github_login) = ?", strings.ToLower(ghUser.GetLogin())` will be sufficient to handle all the scenarios mentioned below.

```
In DB                       From GH
username small case   == username small case
username mixed case   == username mixed case
username small case   == username mixed case
username mixed case   == username small case